test/functional/nulldummy.py

```diff
@@ -40,7 +40,7 @@ class NULLDUMMYTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
-        self.extra_args = [['-whitelist=127.0.0.1', '-walletprematurewitness']]
+        self.extra_args = [['-whitelist=127.0.0.1']]
```

Comment:
the `-whitelist` argument isn't actually needed here. Just delete the line entirely.